docs/microservices/users.md

```diff
@@ -2,66 +2,19 @@
-            models/
-        interfaces/
-        routes/
-        util/
-        validators/
-        app.ts
-        server.ts
-    test/                    .............     # unit tests        
-        controllers/
-    Dockerfile          
-    docker-compose.yaml
-    docker-compose.test.yaml
-    LICENSE
-    nodemon.json
-    package.json
-    package-lock.json
-    swaggerDoc.js
-    tsconfig.json
-    tslint.json
-
-## Getting Started
-
-### For Development
-
-**Requirements**
-
-* [`gcloud`](https://cloud.google.com/sdk/install)
-* [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-* [`docker`](https://www.docker.com/)
-* [`docker-compose`](https://docs.docker.com/compose/)
-
-**Available Commands**
-
-| Command | Description |
-| --- | --- |
-| `npm start` | spins up a MongoDB instance as well as your API server through docker-compose (see docker-compose.yaml). Be sure that ports 27017 and 3000 are unused by other services |
-| `npm run start:local` | runs a nodemon server that listens for changes in the src directory (requires nodemon and ts-node) |
-| `npm run start:prod` | starts the API server in the production env. Must be run after `npm run build` (requires node and tsc) |
-| `npm run test` | runs unit tests on new instances of MongoDB and the API (see docker-compose.test.yaml). Once the tests finish running, both instances stop 
-| `npm run test:local` | runs unit tests without docker and MongoDB (requires nyc, mocha, and ts-node) |
-| `npm run build` | builds the API for production and outputs the result in dist/ (requires tsc) |
-| `npm run lint` | lints the src directory according to tslint.json and tsconfig.json |
-
+## For Development
 
-**Running**
+**Running locally**
 
-NOTE: steps 1 through 3 can be skipped by running a local copy of the problems microservice with docker or node (keep in mind that the default mongodb url for the problems microservice is mongodb://mongo:27017)
+NOTE: steps 1 through 3 can be skipped by running a local copy of the problems microservice with `docker` or `node` (keep in mind that the default mongodb url for the problems microservice is `mongodb://mongo:27017`).
 
-1. Login to a gcloud account that has access to our cluster on GKE
+1. Login to a [`gcloud`](https://cloud.google.com/sdk/docs/quickstarts) account using the terminal tool that has access to our cluster on GKE
 
-2. Connect to our wisp-prod cluster
+2. Connect to our wisp-prod cluster using `kubectl`
+    - `gcloud container clusters get-credentials <cluster-name> --zone <zone-name> --project <project-name>`
 
 3. Use port forwarding to forward traffic on port 3001 to the problems microservice (this will keep running in foreground by default)
+    - Use commands from [this file](https://github.com/Compete-McGill/wisp/blob/master/build/Makefile) to port-forward
 
 4. Clone the [repository](https://github.com/Compete-McGill/wisp-users-microservice), and navigate to the root of the project.
 
```

Comment:
no longer necessary since the docker-compose file got updated. can be changed to the same as the problems microservice (npm install then npm start)

Reply:
Ok, moved this section to the General page then